contracts/libraries/VaultLib.sol

```diff
@@ -266,9 +269,42 @@ library VaultLibrary {
 
         self.vault.lv.issue(from, amount);
 
+        self.vault.userLvBalance[from].balance += amount;
         received = amount;
     }
 
+    // returns the amount of PA user able to redeem
+    function redeemablePA(State storage self, address user) external view returns (uint256) {
+        return self.vault.userLvBalance[user].balance;
+    }
```

Comment:
this should return the total amount of PA that all user combined can redeem, in `State.sol` ->  `VaultWithdrawalPool` -> `paBalance`

```solidity
struct VaultWithdrawalPool {
    uint256 atrributedLv;
    uint256 raExchangeRate;
    uint256 paExchangeRate;
    uint256 raBalance;
    // this one
    uint256 paBalance;
    // this should be removed in the future, as removing it now would break struct encoding
    uint256 stagnatedPaBalance;
}
```